### PR TITLE
Switching token to status field

### PR DESCRIPTION
**Pull request by Kidswiss**
The token should be automagically created for the user. The only thing
relevant to configure is how long the token should be valid for and then
be written to the statusfield.

### CODE REVIEW

**Comment by Kidswiss** [comment]
There may follow more things as I progress through the implementation.

File: docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc [hunk 1]
```diff
@@ -370,15 +371,16 @@ spec:
...
   facts:
     distribution: openshift4
     cloud: aws
     region: eu-west-1
 status:
-  bootstrapTokenValid: true
+  bootstrapToken:
+    token: sdaf445ftzzt
+    validUntil: 2020-01-27 12:00
+   bootstrapTokenValid: true
```
Comment (srueg) [inline docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc]:
```suggestion
    bootstrapTokenValid: true
```

**Review by srueg — DISMISSED** [review]

File: docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc [hunk 2]
```diff
@@ -341,15 +341,16 @@ spec:
...
   facts:
     distribution: openshift3
     cloud: cloudscale
     region: rma1
 status:
-  bootstrapTokenValid: true
+  bootstrapToken:
+    token: sdaf445ftzzt
+    validUntil: 2020-01-27 12:00
+    bootstrapTokenValid: true
```
Comment (tobru) [inline docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc]:
```suggestion
    valid: true
```

File: docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc [hunk 3]
```diff
@@ -370,15 +371,16 @@ spec:
...
   facts:
     distribution: openshift4
     cloud: aws
     region: eu-west-1
 status:
-  bootstrapTokenValid: true
+  bootstrapToken:
+    token: sdaf445ftzzt
+    validUntil: 2020-01-27 12:00
+    bootstrapTokenValid: true
```
Comment (tobru) [inline docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc]:
```suggestion
    valid: true
```

**Pull request merged**